docs/src/lib/interfaces.md

```diff
@@ -74,9 +74,9 @@ surface(::LazySet{N}) where {N}
 area(::LazySet{N}) where {N}
 concretize(::LazySet)
 complement(::LazySet)
-project(S::LazySet{N}, ::AbstractVector{Int}, ::Nothing, ::Int=dim(S)) where {N}
-project(S::LazySet, ::AbstractVector{Int}, ::Type{<:LazySet}, ::Int=dim(S))
-project(S::LazySet, ::AbstractVector{Int}, ::Pair{<:UnionAll, <:Real}, ::Int=dim(S))
+project(S::LazySet{N}, ::AbstractVector{Int}, ::Nothing=nothing, ::Int=dim(S)) where {N}
+project(S::LazySet, ::AbstractVector{Int}, ::Type{var"#s27"} where var"#s27"<:LazySet, ::Int=dim(S))
```

Comment:
that hmm not robust? if we define a new variable in that test (or a new lazyset type?) that may change

Reply:
What do you mean?

Reply:
that `var"#s27"` refers to some internal id that may change.

Reply:
Hm, but at least it works for now. I could not find a different way. What should we do?

Reply:
ok